src/bitcoin-cli.cpp

```diff
@@ -991,6 +1057,11 @@ static int CommandLineRPC(int argc, char *argv[])
                 if (gArgs.IsArgSet("-getinfo") && !gArgs.IsArgSet("-rpcwallet")) {
                     GetWalletBalances(result); // fetch multiwallet balances and append to result
                 }
+
+                if (gArgs.IsArgSet("-getinfo")) {
```

Comment:
nit, after using `if (gArgs.IsArgSet("-getinfo"))` a year ago, I learned later that this does more robust value checking of the user input:
```suggestion
                if (gArgs.GetBoolArg("-getinfo", false)) {
```

Reply:
Updated to use `GetBoolArg` instead of `IsArgSet`